doc/api/cli.md

```diff
@@ -2464,6 +2464,54 @@ added:
 Configures the test runner to exit the process once all known tests have
 finished executing even if the event loop would otherwise remain active.
 
+### `--test-global-setup=module`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Specify a module that will be run before all tests are executed and
```

Comment:
```suggestion
Specify a module that will be evaluated before all tests are executed and
```

I mean the `globalTeardown` will be run after, so I'm not sure (I'm also not sure evaluated is correct)